tensorflow/lite/micro/micro_profiler.h

```diff
@@ -56,15 +64,19 @@ class MicroProfiler {
   // Prints the profiling information of each of the events.
   void Log() const;
 
+  // Returns the number of active events that are being profiled.
+  uint32_t GetTotalEvents() const;
```

Comment:
nit: have this return an int instead of uint32_t?

Reply:
Done.